packages/calling-component-bindings/src/providers/CallClientProvider.tsx

```diff
@@ -4,18 +4,32 @@
 import { StatefulCallClient, StatefulDeviceManager } from '@internal/calling-stateful-client';
 import React, { createContext, useContext, useEffect, useState } from 'react';
 
+/**
+ * @private
+ */
 export type CallClientContextType = {
   callClient: StatefulCallClient;
   deviceManager: StatefulDeviceManager | undefined;
 };
 
+/**
```

Comment:
These should be exposed to enable people getting call/callAgent when they start with component level